src/crimson/os/seastore/seastore.cc

```diff
@@ -898,7 +898,7 @@ SeaStore::_omap_list_ret SeaStore::_omap_list(
     root,
     start,
     [&t, config](auto &manager, auto& root, auto& start) {
-      return manager.omap_list(root, t, start, config);
+      return manager.omap_list(root, t, start, std::nullopt, config);
```

Comment:
wrap `end` with do_with.

Reply:
There is no param `end` in `omap_list`, do i need to set a default param `end = std::nullopt` outside or write like this: do_with(std::nullopt, [](end){})

Reply:
I meant `end/last`. Similar to `start/first`, it is captured as reference in the internal implementation, so need caller to keep it alive during the call.

Reply:
It will be better to also align the variable names.